src/devices/sber.ts

```diff
@@ -0,0 +1,60 @@
+import {Definition} from '../lib/types';
+import {battery, humidity, iasZoneAlarm, ignoreClusterReport, temperature} from '../lib/modernExtend';
+import {modernExtend as tuyaModernExtend} from '../lib/tuya';
+const {tuyaMagicPacket, tuyaOnOffActionLegacy} = tuyaModernExtend;
+
+const definitions: Definition[] = [
+    {
+        fingerprint: [{modelID: 'SM0202', manufacturerName: '_TYZB01_2jzbhomb'}],
+        model: 'SBDV-00029',
+        vendor: 'Sber',
+        description: 'Smart motion sensor',
+        extend: [
+            iasZoneAlarm({zoneType: 'occupancy', zoneAttributes: ['alarm_1', 'tamper', 'battery_low'], alarmTimeout: true}),
+            battery({voltage: true, voltageReporting: true}),
+        ],
+    },
+    {
+        fingerprint: [{modelID: 'TS0203', manufacturerName: '_TYZB01_epni2jgy'}],
+        model: 'SBDV-00030',
+        vendor: 'Sber',
+        description: 'Smart opening sensor',
+        extend: [
+            ignoreClusterReport({cluster: 'genBasic'}),
+            iasZoneAlarm({zoneType: 'contact', zoneAttributes: ['alarm_1', 'tamper', 'battery_low']}),
+            battery({voltage: true, voltageReporting: true}),
+        ],
+    },
+    {
+        fingerprint: [{modelID: 'TS0041A', manufacturerName: '_TYZB01_ub7urdza'}],
+        model: 'SBDV-00032',
+        vendor: 'Sber',
+        description: 'Smart button',
+        extend: [
+            tuyaMagicPacket(),
+            tuyaOnOffActionLegacy({actions: ['single', 'double', 'hold']}),
```

Comment:
Since this is a new device, why not use `tuyaOnOffAction`?

Reply:
It's not new, I pulled it out of tuya whitelabels list.